cloud/manifest_reader.h

```diff
@@ -26,12 +26,20 @@ class ManifestReader {
   // Retrieve all live files referred to by this bucket path
   Status GetLiveFiles(const std::string bucket_path, std::set<uint64_t>* list) {
     std::unique_ptr<CloudManifest> cloud_manifest;
-    return GetLiveFilesAndCloudManifest(bucket_path, list, &cloud_manifest);
+    // TODO(wei): this is not correct if we can roll cloud manifest without
+    // reopening the shard. We should fix this by reading from
+    // cenv_->cloud_manifest directly
+    return GetLiveFilesAndCloudManifest(
+        std::move(bucket_path), cenv_->GetCloudEnvOptions().cookie_on_open,
```

Comment:
`std::move(bucket_path)` doesn't move because `bucket_path` is const. See below, too.